config/environments/development.rb

```diff
@@ -42,7 +42,7 @@
   config.paperclip_defaults = {
     :storage => :s3,
     :s3_credentials => {
-      :bucket => ENV['MY_BUCKET_NAME'],
+      :bucket => ENV['S3_BUCKET_NAME'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.